sonnet/src/optimizers/momentum.py

```diff
@@ -87,8 +87,7 @@ def apply(self, updates: Sequence[types.ParameterUpdate],
     And when using Nesterov momentum (`use_nesterov=True`) it applies:
 
         accum_t <- momentum * accum_{t-1} + update
-        parameter <- parameter - (learning_rate * update +
-                                  learning_rate * momentum * accum_t)
+        parameter <- parameter - (learning_rate * update + learning_rate * momentum * accum_t)
```

Comment:
OOC, is this change necessary?

Reply:
Those lines showed unexpected indentation warning. If you don't like it, how about this? 623a88d